package.json

```diff
@@ -6,6 +6,8 @@
     "nx": "nx",
     "start": "nx serve",
     "build": "nx build",
+    "build:ci:server": "nx build server --skip-nx-cache --prod", 
+    "build:ci:client": "nx build client --skip-nx-cache --prod",
```

Comment:
Added these two, because `nx build` liked to always try to build the client when I passed `--skip-nx-cache` to it. Decided to just make it explicit.

And we _might_ be able to use the nx cache in the future, but after looking at the docs, it's complicated enough that I dunno if we want to bother.